### PR TITLE
python3.pkgs.cve-bin-tool: init at unstable-2021-04-15

### PR DESCRIPTION
**Pull request by cole-h**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

----

~~Waiting on next staging-next merge for python3.pkgs.py 1.10.0 and python3.pkgs.pyyaml 5.4.1.~~

### CODE REVIEW

File: pkgs/development/python-modules/cve-bin-tool/default.nix [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+, rich
+, aiohttp
+, toml
+  # aiohttp[speedups]
+, aiodns
+, brotlipy
+, cchardet
+, pillow
+}:
+buildPythonPackage {
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/cve-bin-tool/default.nix:27]:
```suggestion
}:

buildPythonPackage {
```

File: pkgs/development/python-modules/cve-bin-tool/default.nix [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+    zstandard
+    rich
+    aiohttp
+    toml
+
+    # aiohttp[speedups]
+    aiodns
+    brotlipy
+    cchardet
+    pillow # needed by brotlipy
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/cve-bin-tool/default.nix:64]:
```suggestion
     # needed by brotlipy
    pillow
```

Comment (SuperSandro2000) [inline pkgs/development/python-modules/cve-bin-tool/default.nix:64]: Do we want to move those into brotlipy and aiohttp? I would be in favor of that especially for brotlipy.

File: pkgs/development/python-modules/cve-bin-tool/default.nix [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+  ];
+
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/cve-bin-tool/default.nix:66]: missing pythonImportsCheck.

File: pkgs/development/python-modules/rpmfile/default.nix [hunk 4]
```diff
@@ -0,0 +1,27 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, setuptools_scm
+}:
+buildPythonPackage rec {
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/rpmfile/default.nix:6]:
```suggestion
}:

buildPythonPackage rec {
```

File: pkgs/development/python-modules/rpmfile/default.nix [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "e56cfc10e1a7d953b1890d81652a89400c614f4cdd9909464aece434d93c3a3e";
+  };
+
+  # Tests access the internet
+  doCheck = false;
+
+  nativeBuildInputs = [ setuptools_scm ];
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/rpmfile/default.nix:18]:
```suggestion
  nativeBuildInputs = [ setuptools-scm ];
```

File: pkgs/development/python-modules/rpmfile/default.nix [hunk 6]
```diff
@@ -0,0 +1,27 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, setuptools_scm
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/rpmfile/default.nix:4]:
```suggestion
, setuptools-scm
```

File: pkgs/development/python-modules/rpmfile/default.nix [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+  nativeBuildInputs = [ setuptools_scm ];
+
+  meta = with lib; {
+    description = "Read rpm archive files";
+    homepage = "https://github.com/srossross/rpmfile";
+    license = licenses.mit;
+    maintainers = teams.determinatesystems.members;
+  };
+}
+
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/rpmfile/default.nix:27]: One final new line to much.

File: pkgs/development/python-modules/rpmfile/default.nix [hunk 8]
```diff
@@ -0,0 +1,27 @@
...
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "e56cfc10e1a7d953b1890d81652a89400c614f4cdd9909464aece434d93c3a3e";
+  };
+
+  # Tests access the internet
+  doCheck = false;
+
+  nativeBuildInputs = [ setuptools_scm ];
+
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/rpmfile/default.nix:19]: missing pythonImportsCheck.

**Comment by SuperSandro2000** [comment]
We can also merge it into staging-next to cheat the waiting time a bit.

**Comment by cole-h** [comment]
@ofborg build python3.pkgs.cve-bin-tool

**Pull request merged**